picard/track.py

```diff
@@ -158,6 +158,13 @@ def _convert_folksonomy_tags_to_genre(self):
             genre = [join_tags.join(genre)]
         self.metadata['genre'] = genre
 
+    def _get_ignored_folksonomy_tags(self):
+        tags = []
+        ignore_tags = config.setting['ignore_tags']
+        if ignore_tags:
+            tags = [s.strip().lower() for s in ignore_tags.split(',')]
+        return tags
+
```

Comment:
You just left out the `ìf config.setting['ignore_tags']`, and I added that since I was not sure whether we can rely on this value always being a string. If it is e.g. `None` your code will fail.